homeassistant/components/swiss_public_transport/strings.json

```diff
@@ -2,34 +2,31 @@
   "config": {
     "error": {
       "cannot_connect": "Cannot connect to server",
-      "bad_config": "Request failed due to bad config: Check at [stationboard](http://transport.opendata.ch/examples/stationboard.html) if your station names are valid",
+      "bad_config": "Request failed due to bad config: Check at [stationboard]({stationboard_url}) if your station names are valid",
       "unknown": "An unknown error was raised by python-opendata-transport"
     },
     "abort": {
-      "already_configured": "[%key:common::config_flow::abort::already_configured_service%]",
```

Comment:
Why are they not needed?

Reply:
Because these aborts only happen on import, and we don't show them to people

Reply:
That doesn't mean we shouldn't add them...

If they are used in the config flow, they should be part of the strings.